### PR TITLE
Add instructions to DANDI api_key input

### LINKED ISSUE #339: Add instructions to DANDI API key area of settings page
Not a huge priority; but a first-time user of the GUIDE might not know how to fetch/generate their DANDI API Token 

Some small instructions around that area of the setting page could be very helpful

### PR DESCRIPTION
**Pull request by garrettmflynn**
fix #339

### CODE REVIEW

File: schemas/json/dandi/global.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "properties": {
         "api_key": {
             "type": "string",
-            "format": "password"
+            "format": "password",
+            "description": "Log in to DANDI, click on your user initials in the top-right corner, and copy your API key from the resulting pop-up. <br/><small><b>Note:</b> <a href='https://dandiarchive.org' target='_blank'>Production</a> and <a href='https://gui-staging.dandiarchive.org'  target='_blank'>staging</a> servers have different API keys and different logins</small>"
```
Comment (CodyCBakerPhD) [inline schemas/json/dandi/global.json:6]:
```suggestion
            "description": "Log in to DANDI, click on your user initials in the top-right corner, and copy your API key from the resulting pop-up. <br/><small><b>Note:</b> <a href='https://dandiarchive.org' target='_blank'>The main archive</a> and <a href='https://gui-staging.dandiarchive.org'  target='_blank'>the staging (testing) server</a> have different API keys and different logins</small>"
```

**Comment by CodyCBakerPhD** [comment]
Overall, exactly what I was going for - looks great!

One minor rephrasing of the differentiation between the two DANDI servers to match how they are usually described to new users

**Comment by garrettmflynn** [comment]
Perfect. Ready to merge?

**Comment by CodyCBakerPhD** [comment]
Yep, will merge once #337 is into main

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn Looks like this PR (or the recent changes to `main`) have broken many of the Chromatic stories

**Comment by garrettmflynn** [comment]
The merge was incorrect. I've fixed it now.

**Comment by CodyCBakerPhD** [comment]
> The merge was incorrect. I've fixed it now.

Whoops, my bad on that one - thought the description was a part of this branches changes

**Pull request merged**